OrbitGl/SamplingReport.cpp

```diff
@@ -43,6 +43,11 @@ void SamplingReport::FillReport() {
 }
 
 void SamplingReport::UpdateDisplayedCallstack() {
+  if (selected_address_ == SamplingReport::kInvalidFunctionAddress) {
+    ClearReport();
```

Comment:
Ah... `ClearReport()` only resets the selected callstack, it doesn't clear the complete report? The name is very misleading and I completely misread that, but then this makes sense.

Reply:
I change to select function address as kInvalidFunctionAddress to indicate no function is selected when moving to a thread tab without selections. So the action of clearing the callstack report is move from the orbitsamplingreport.cpp to the SamplingReport.cpp, which is closer to the definition of ClearReport. Hope it would be less misleading.